### ISSUE TITLE
Doesn't work on M1 Mac, missing /usr/bin/python

### ISSUE THREAD

**Issue by ferrari** [vercel/hyper#6429]
- [ ] I am on the [latest](https://github.com/vercel/hyper/releases/latest) Hyper.app version
- [ ] I have searched the [issues](https://github.com/vercel/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Monterey
- **Hyper.app version**: 3.2.0 Stable
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/ferrari/63fe9c9022d455c48d93d167aafef685

- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

I'm on a M1 Macbook Pro, and they do not have the native python installed. When i run the `hyper` command on CLI.

I get 

```
/usr/local/bin/hyper: line 4: /usr/bin/python: No such file or directory
/usr/local/bin/hyper: line 8: ./MacOS/Hyper: No such file or directory
```

I've tried creating a symlink to my existing Python3 folder, to no avail. I guess there's something I don't know or maybe i've done something wrong.

**Comment by andrewconnell** [comment]
Same issue with a clean install of macOS 12.3 (Monterey) non-Apple silicon (M1)

Installed Hyper with homebrew.

**Comment by rjgux** [comment]
Same problem here.  macOS 12.3 (Monterey) Apple silicon (M1)

**Comment by jabr** [comment]
A quick, manual fix: edit the `hyper` command and replace `/usr/bin/python` with `/usr/bin/python3`.

You can find the location of the command with `which hyper`. It is likely at `/opt/homebrew/bin/hyper` or `/usr/local/bin/hyper`.

**Comment by unshade** [comment]
I just modified it, unfortunately it doesn't change much, the second error is still present, and the script runs and displays errors in a loop. Attached is a screenshot

<img width="590" alt="image" src="https://user-images.githubusercontent.com/72690981/161122562-5373ceb0-2061-4fad-98c7-cc63cf0334a6.png">

**Comment by jabr** [comment]
> I just modified it, unfortunately it doesn't change much, the second error is still present, and the script runs and displays errors in a loop.

It looks like your editor might have word-wrapped a line an introduced a newline. Does the script look like this?
```
function realpath() { /usr/bin/python3 -c "import os,sys; print(os.path.realpath(sys.argv[1]))" 
"$0"; }
```
And not this?
```
function realpath() { /usr/bin/python3 -c "import os,sys; print(os.path.realpath(sys.argv[1]))" "$0"; }
```

**Comment by rjgux** [comment]
Thank you @jabr!

**Comment by DDeme** [comment]
I have a same problem. Hyper works but I can`t start it with command.

**Comment by gwenwindflower** [comment]
same issue, thanks for the tip!

**Comment by LabhanshAgrawal** [comment]
Fixed in 3.2.1
